.github/workflows/ci.yml

```diff
@@ -37,3 +40,11 @@ jobs:
 
       - name: Unit Tests
         run: composer t3g:test:php:unit
+
+      - name: Functional Tests
+        run: php .build/bin/phpunit -c Build/FunctionalTests.xml --colors;
```

Comment:
can we move this to a composer command please, ddev setup should also be adjusted so we can run them locally